crates/server/src/store/recursor/config.rs

```diff
@@ -61,6 +61,10 @@ pub struct RecursiveConfig {
     #[serde(default)]
     pub do_not_query: Vec<IpNet>,
 
+    /// Automatically add recommended filters to the do not query list
+    #[serde(default = "recommended_filters_default")]
+    pub recommended_filters: bool,
```

Comment:
Instead of adding this field, should we use a custom default for `do_not_query` instead? This seems potentially confusing.